CHANGELOG_iOS.md

```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## [1.70.131](https://github.com/brave/brave-browser/releases/tag/v1.70.131)
+
+### Web3
+
+ - Updated ‘Confirm Transactions’ page to display contract address as hyperlink. ([#39720](https://github.com/brave/brave-browser/issues/39720))
+
+### General
+
+ - Implemented search for ‘Content Filtering’ settings page. ([#36036](https://github.com/brave/brave-browser/issues/36036))
+ - Added option to force update filter lists in ‘Content Filtering’ settings page. ([#35982](https://github.com/brave/brave-browser/issues/35982))
+ - Added support for Dark Mode and Tinted icons. ([#41088](https://github.com/brave/brave-browser/issues/41088))
+ - Added Leo to the Home Screen shortcuts widget. ([#39886](https://github.com/brave/brave-browser/issues/39886))
+ - Added support for modifying user prompt in Leo. ([#39831](https://github.com/brave/brave-browser/issues/39831))
+ - Added skipping debouncing if canonicalized URL’s hostname doesn’t match unescaped URLs hostname. ([#39866](https://github.com/brave/brave-browser/issues/39866))
+ - Added ability to edit custom search engines after being added in ‘Search Engines’ settings page. ([#39552](https://github.com/brave/brave-browser/issues/39552))
+ - Added DarkReader support when Night Mode is enabled. ([#39786](https://github.com/brave/brave-browser/issues/39786))
+ - Added handling potential infinite redirect loop with HTTPS upgrades. ([#40346](https://github.com/brave/brave-browser/issues/40346))
+ - Added right-to-left (RTL) language support. ([#40471](https://github.com/brave/brave-browser/issues/40471))
+ - Updated ’Manage Website Data’ to remove ads data as well when clearing browsing history. ([#39051](https://github.com/brave/brave-browser/issues/39051))
+ - Updated icon for starting a new conversation in Leo. ([#39818](https://github.com/brave/brave-browser/issues/39818))
+ - Updated input box to be in focus when Leo in opened. ([#39858](https://github.com/brave/brave-browser/issues/39858))
```

Comment:
`is` instead of `in`.

```
 - Updated input box to be in focus when Leo is opened. ([#39858](https://github.com/brave/brave-browser/issues/39858)
```

Reply:
Thanks, good catch.